src/app.module.ts

```diff
@@ -2,10 +2,29 @@ require('dotenv').config();
 import { Module } from '@nestjs/common';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
+import { UserModule } from './users/user.module';
+import { Connection } from 'typeorm';
+import { TypeOrmModule } from '@nestjs/typeorm';
+
+import { User } from './users/user.entity';
 
 @Module({
-  imports: [],
+  imports: [
+    TypeOrmModule.forRoot({
+      type: 'postgres',
+      host: process.env.DB_HOST,
+      port: Number(process.env.DB_PORT),
+      username: process.env.DB_USER,
+      password: process.env.DB_PASS,
+      database: process.env.DB_NAME,
+      entities: [User],
```

Comment:
@haltaction we have to add every entity here, isn't it?

Reply:
For now yes, it's fine. In the future, this block could be moved to a separate module, e.g. _DatabaseModule_ 
And in _app.module_ we could only import this module.

Reply:
Also there is a way for many entities - use `"entities": ["dist/**/*.entity{.ts,.js}"],` https://docs.nestjs.com/techniques/database#typeorm-integration